src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -241,6 +241,43 @@ def capacity(self) -> int:
         """
         return self._buffer.maxlen
 
+    def at(self, key: int | datetime) -> float:  # pylint: disable=invalid-name
```

Comment:
```suggestion
    # pylint before 3.0 only accepts names with 3 or more chars
    def at(self, key: int | datetime) -> float:  # pylint: disable=invalid-name
```

Maybe add an extra clarification about this, as it is not obvious that `pylint` is complaining about the length. Also when googling for the error the first result is the documentation for the upcoming 3.0 that mentions this, but if you go to the 2.x documentation it doesn't even mention there is a minimum length.